tutorials/lava/lib/peripherals/dvs/PropheseeCamera.ipynb

```diff
@@ -0,0 +1,313 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "21d15919-bad4-4363-b458-520b30510ca5",
+   "metadata": {},
+   "source": [
+    "# Prophesee Event Camera \n",
+    "\n",
+    "In this tutorial we demonstrate how to obtain data from a Prophesee camera and stream the events into Lava. We show the basic usage of the PropheseeCamera Process and how to apply filters and transformations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "1df6eded-4456-406f-b5b9-bab9bc7c687a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/usr/lib/python3/dist-packages/metavision_core_ml/preprocessing/viz.py:41: NumbaDeprecationWarning: \u001b[1mThe 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.\u001b[0m\n",
+      "  def _viz_events(events, img):\n"
+     ]
+    }
+   ],
+   "source": [
+    "import matplotlib\n",
+    "%matplotlib inline\n",
+    "\n",
+    "from lava.magma.core.run_configs import Loihi2SimCfg\n",
+    "from lava.magma.core.run_conditions import RunSteps\n",
+    "\n",
+    "from lava.lib.peripherals.dvs.prophesee import PropheseeCamera\n",
+    "from metavision_core.utils import get_sample\n",
+    "from metavision_sdk_cv import TrailFilterAlgorithm, ActivityNoiseFilterAlgorithm\n",
+    "from metavision_core.event_io import RawReader\n",
+    "from lava.lib.peripherals.dvs.transform import Compose, Downsample\n",
+    "\n",
+    "from utils import VisProcess"
```

Comment:
Could you rename `VisProcess` into `EventVisualizer' or something similar? There is no need for abbreviating the name and adding the `Process` suffix is not necessary.